components/zcash_protocol/src/constants/regtest.rs

```diff
@@ -13,34 +13,42 @@ pub const COIN_TYPE: u32 = 1;
 ///
 /// It is defined in [the `zcashd` codebase].
 ///
-/// [`ExtendedSpendingKey`]: crate::sapling::zip32::ExtendedSpendingKey
+/// [`ExtendedSpendingKey`]: https://docs.rs/sapling-crypto/latest/sapling_crypto/zip32/struct.ExtendedSpendingKey.html
 /// [the `zcashd` codebase]: <https://github.com/zcash/zcash/blob/128d863fb8be39ee294fda397c1ce3ba3b889cb2/src/chainparams.cpp#L496>
 pub const HRP_SAPLING_EXTENDED_SPENDING_KEY: &str = "secret-extended-key-regtest";
 
 /// The HRP for a Bech32-encoded regtest Sapling [`ExtendedFullViewingKey`].
 ///
 /// It is defined in [the `zcashd` codebase].
 ///
-/// [`ExtendedFullViewingKey`]: crate::sapling::zip32::ExtendedFullViewingKey
+/// [`ExtendedFullViewingKey`]: https://docs.rs/sapling-crypto/latest/sapling_crypto/zip32/struct.ExtendedFullViewingKey.html
 /// [the `zcashd` codebase]: <https://github.com/zcash/zcash/blob/128d863fb8be39ee294fda397c1ce3ba3b889cb2/src/chainparams.cpp#L494>
 pub const HRP_SAPLING_EXTENDED_FULL_VIEWING_KEY: &str = "zxviewregtestsapling";
 
 /// The HRP for a Bech32-encoded regtest Sapling [`PaymentAddress`].
 ///
 /// It is defined in [the `zcashd` codebase].
 ///
-/// [`PaymentAddress`]: crate::sapling::PaymentAddress
+/// [`PaymentAddress`]: https://docs.rs/sapling-crypto/latest/sapling_crypto/struct.PaymentAddress.html
 /// [the `zcashd` codebase]: <https://github.com/zcash/zcash/blob/128d863fb8be39ee294fda397c1ce3ba3b889cb2/src/chainparams.cpp#L493>
 pub const HRP_SAPLING_PAYMENT_ADDRESS: &str = "zregtestsapling";
 
+/// The prefix for a Base58Check-encoded regtest Sprout address.
+///
+/// Defined in the [Zcash Protocol Specification section 5.6.3][sproutpaymentaddrencoding].
+/// As is specified, this is identical to the testnet prefix.
```

Comment:
```suggestion
/// Same as the testnet prefix.
```
Regtest is not specified in the protocol spec.